sklearn/manifold/_mds.py

```diff
@@ -426,11 +502,21 @@ def fit_transform(self, X, y=None, init=None):
             raise ValueError("Proximity must be 'precomputed' or 'euclidean'."
```

Comment:
While we're at it, let's make this old error message correct

```suggestion
            raise ValueError("dissimilarity must be 'precomputed' or 'euclidean'."
```

Reply:
Done